flamingpy/codes/graphs/egraph.py

```diff
@@ -204,6 +204,12 @@ def draw(self, **kwargs):
 
         _, ax = draw_EGraph(self, **kwargs)
         return ax
+    
+    def draw_3DScatterPlot(self,**kwargs):
+        from flamingpy.utils.viz import draw_EGraph_3DScatterPlot
```

Comment:
Please move this one to the start of the file like the other imports

Reply:
I'm sorry, but I don' understand what you mean by moving this to the start of the file. Can you please explain?

Reply:
Oh sorry, my bad. What you did is fine as it's similar to the existing `draw` method 🙈

Reply:
No problem :) Well, I hope to push the final changes soon(regardless of the unitaryhack deadline).